hazelcast/src/main/java/com/hazelcast/spi/properties/GroupProperty.java

```diff
@@ -217,6 +217,32 @@
     public static final HazelcastProperty IO_OUTPUT_THREAD_COUNT
             = new HazelcastProperty("hazelcast.io.output.thread.count", IO_THREAD_COUNT);
 
+    /**
+     * Optimization that allows sending of packets over the network to be done on the calling thread if the
+     * conditions are right. This can reduce latency and increase performance for low threaded environments.
+     *
+     * It is enabled by default because Hazelcast will suppress write through as long as Hazelcast detects
```

Comment:
It's disabled by default: `new HazelcastProperty("hazelcast.io.write.through", false)`

Reply:
Correct. This is intentional.

Reply:
But javadoc says it's enabled by default?

Reply:
That was not intentional :)